frontend/javascripts/oxalis/model/sagas/volume/volume_interpolation_saga.ts

```diff
@@ -389,8 +428,8 @@ export default function* maybeInterpolateSegmentationLayer(): Saga<void> {
 
   for (let u = 0; u < size[firstDim]; u++) {
     for (let v = 0; v < size[secondDim]; v++) {
-      const firstVal = firstSliceDists.get(u, v);
-      const lastVal = lastSliceDists.get(u, v);
+      const firstVal = Number(firstSliceDists.get(u, v));
+      const lastVal = Number(lastSliceDists.get(u, v));
```

Comment:
Not that important, but isn't `firstSlice` and, therefore, `firstSliceDists` of type `NdArray<TypedArrayWithoutBigInt>`. I would have expected the `Number()` call to be superfluous.

Reply:
you're right. this change also originates from an older version where big ints were allowed here, too. changing it now.